### ISSUE TITLE
False positive when using usesnapshot

### ISSUE THREAD

**Issue by la55u** [pmndrs/eslint-plugin-valtio#35]
I get the following warning on the `snap` variable in the condition when trying to use the snapshot to conditionally render some component: 
`Better to just use proxy state.`

Example reproduction:
```
export const Test = () => {
  const snap = useSnapshot(state)
  if(!snap) return null;
  return(
    <></>
  )
}
```

valtio: 1.7.0
eslint-plugin-valtio: 0.6.0

**Comment by dai-shi** [comment]
@barelyhuman can take a look?

**Comment by barelyhuman** [comment]
I've already fixed this in my local patch of #34 as well, the next patch should work for you. 
Thanks for the patience

**Comment by barelyhuman** [comment]
Appreciate the patience, 
can you confirm that the latest build from the 

> Local Build Instructions 

from the following [link](https://ci.codesandbox.io/status/pmndrs/eslint-plugin-valtio/pr/34/builds/299339) solves your issue ?

**Comment by la55u** [comment]
Works for me, thank you!

**Comment by barelyhuman** [comment]
Fixed in v0.6.1
